hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsInputStreamStatistics.java

```diff
@@ -100,28 +100,31 @@ public void testSeekStatistics() throws IOException {
     AbfsOutputStream out = null;
     AbfsInputStream in = null;
 
+    int readBufferSize = getConfiguration().getReadBufferSize();
```

Comment:
Maybe we can reduce the diff on the test by reducing the bufferSize on the filesystem instance so that rest of the asserts remain valid.

Reply:
modified to use original buffer and reduced readBufferSize